S2VX.Game/Editor/CommandPanel/CommandPanelValueInput.cs

```diff
@@ -23,6 +43,10 @@ public class CommandPanelValueInput : Container {
             Position = new(0, S2VXCommandPanel.InputSize.Y)
         };
 
+        public Func<double> CurrentTimeDelegate { get; private set; }
+
+        public void UseApplyCurrentTime() => BtnApplyCurrentTime.Show();
```

Comment:
I wouldn't explicitly call `UseApplyCurrentTime` to show the time button. I can see that it's following the way `UseColorPicker` is written, but `UseColorPicker` was written that particular way it was because when you switch to a different type of command, you have to decide at that moment whether to show/hide the color picker. The time button is different because you know at construction time whether to show or hide this button. I would put the show/hide time button logic into the constructor. You can just check if the passed in delegate is null or not.

Reply:
Implemented